packages/client/test/miner/miner.spec.ts

```diff
@@ -101,7 +102,7 @@ const consensusConfig = {
     epoch: 30000,
   },
 }
-const defaultChainData = {
+const defaultChainData: Omit<GethGenesis, 'alloc'> = {
```

Comment:
Same nit here: should we default `alloc = {}` here instead of omitting it from the type?